packages/sourcecred/src/ui/utils/Web3Context.js

```diff
@@ -0,0 +1,106 @@
+// @flow
+
+import WalletConnectProvider from "@walletconnect/web3-provider";
+import {providers} from "ethers";
+import * as React from "react";
+import Web3Modal from "web3modal";
+
+const INFURA_ID = "a60f8c4d3d4a40a49c4568570a7546b7";
```

Comment:
Created a SourceCred project on Infura under my account. Can replace this with an official sourcecred account if/when we need but the free tier should suffice for a long time

Reply:
Can you provide a little more context on this?

Reply:
Infura is used to connect users to an existing Ethereum node without needing to run a node / browser client themselves. Its pretty much used by every dapp out there, and required by WalletConnect. If we get big enough for us to start hitting the limits, we can change it to require instance maintainers to provide their own Infura ID in config somewhere.